.eslintrc.json

```diff
@@ -0,0 +1,58 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "plugin:react/recommended",
+        "airbnb"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react",
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "no-use-before-define": "off",
+        "@typescript-eslint/no-use-before-define": [
+            "error"
+        ],
+        "import/extensions": "off",
+        "react/prop-types": "off",
+        "react/require-default-props": "off",
+        "react/jsx-filename-extension": [
+            2,
+            {
+                "extensions": [
+                    ".js",
+                    ".jsx",
+                    ".ts",
+                    ".tsx"
+                ]
+            }
+        ],
+        "react/react-in-jsx-scope": "off",
+        "camelcase": "off",
+        "react/jsx-props-no-spreading": "off",
+        "linebreak-style": "off"
```

Comment:
No rules for the use of semicolons and indentation? or is it included in airbnb's rule-set?

Reply:
Yes, that's included.
Most of the rules written in this file are written to align `eslint` with `next.js` and our custom use cases.